src/Database/PostgreSQL/PQTypes/DerivingVia.hs

```diff
@@ -0,0 +1,113 @@
+{-# LANGUAGE AllowAmbiguousTypes, TypeApplications #-}
+module Database.PostgreSQL.PQTypes.DerivingVia (
+  -- | Helpers, to be used with `deriving via`.
+    SQLEnum(..)
+  , SQLEnumEncoding(..)
+  , SQLEnumAsText(..)
+  , SQLEnumAsTextEncoding(..)
+  , isInjective  -- ^ for doctests
```

Comment:
Use `-- *` in a previous line, this syntax breaks haddock.

Reply:
Fixed.